### DISCUSSION TITLE
Mimir ruler throws EOF error when connection to alertmanager

### DISCUSSION THREAD

**Discussion by ItsMisterP** [grafana/mimir#6586] [Help and support]
#### Describe the bug
Hey everyone. I am running mimir on production environment. From time to time i got the following error message from mimir-ruler:

Post "http://ip:8080/mimir/alertmanager/api/v2/alerts": read tcp ip:34530->ip:8080: read: connection reset by peer
and
Post "http://ip:8080/mimir/alertmanager/api/v2/alerts": EOF

Does anyone knows this kind of errors and can help to get rid of this kind of errors.

#### To Reproduce

1. Mimir in multi tenant mode.
#### Expected behavior

None of this error occure.

#### Environment

- Infrastructure: AWS ECS
- Deployment tool: AWS CDK

thank you in advance :)
Philip

**Comment by dimitarvdimitrov** [comment]
this seems like something is interrupting the connection to the alertmanager. Could be the alertmanager crashing upon receiving the request for example? Perhaps ECS doesn't allow this traffic (although I'd expect this manifests in dial timeouts)?

are there any logs from the alertmanager around the times of the requests?

**Reply by ItsMisterP** [reply]
Hey thanks for your response. The Alertmanager doesnt have any logs regarding this issue. 
Is it possible to see which alert rules are causing this issue?
